images-list

```diff
@@ -280,8 +280,10 @@ quay.io/cilium/startup-script rancher/mirrored-cilium-startup-script 62bfbe88c17
 quay.io/coreos/kube-state-metrics rancher/mirrored-coreos-kube-state-metrics v1.9.7
 quay.io/coreos/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.39.0
 quay.io/coreos/prometheus-config-reloader rancher/mirrored-coreos-prometheus-config-reloader v0.38.1
+quay.io/coreos/prometheus-config-reloader rancher/mirrored-coreos-prometheus-config-reloader v0.39.0
 quay.io/coreos/prometheus-operator rancher/coreos-prometheus-operator v0.39.0
 quay.io/coreos/prometheus-operator rancher/mirrored-coreos-prometheus-operator v0.38.1
+quay.io/coreos/prometheus-operator rancher/mirrored-coreos-prometheus-operator v0.39.0
```

Comment:
Hmm, seems like we are using the older quay.io repositories here, but I checked the new ones at `quay.io/prometheus-operator/prometheus-config-reloader` and `quay.io/prometheus-operator/prometheus-operator` and they don't have the v0.39.0 image tags.

Looks good to me, but just wanted to leave this as a comment so, if we update Monitoring V1, we make sure we migrate to the new repositories next time.